### PR TITLE
feat(cli): Add example to argocd relogin command

### PR DESCRIPTION
**Pull request by chiranjeevir-git**
Hi team,
With this PR I would like to address the missing examples for the command "argocd relogin" for issue https://github.com/argoproj/argo-cd/issues/15690.

https://github.com/argoproj/argo-cd/blob/a934772778a041f52675158bedc5a8d3cc313fc2/cmd/argocd/commands/relogin.go#L26

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] The title of the PR conforms to the [Toolchain Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/toolchain-guide/#title-of-the-pr)
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md#legal)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).
* [ ] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
* [ ] I have added a brief description of why this PR is necessary and/or what this PR solves.

### CODE REVIEW

File: docs/user-guide/commands/argocd_relogin.md [hunk 1]
```diff
@@ -15,7 +15,21 @@ argocd relogin [flags]
 ### Examples
 
 ```
-argocd relogin --password YOUR_PASSWORD
+  
+# Reinitiates the login with previous contexts
+argocd relogin
+
+# Reinitiates the logins with sso
+argocd relogin --sso
```
Comment (crenshaw-dev) [inline docs/user-guide/commands/argocd_relogin.md:23]: The `--sso` flag doesn't exist for the `relogin` command

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

File: docs/user-guide/commands/argocd_relogin.md (hunk 1 above)
Comment (chiranjeevir-git) [reply docs/user-guide/commands/argocd_relogin.md]: Thanks for correcting @crenshaw-dev 
updated the PR

File: cmd/argocd/commands/relogin.go [hunk 2]
```diff
@@ -85,6 +84,18 @@ func NewReloginCommand(globalClientOpts *argocdclient.ClientOptions) *cobra.Comm
 			errors.CheckError(err)
 			fmt.Printf("Context '%s' updated\n", localCfg.CurrentContext)
 		},
+		Example: `  
+# Reinitiates the login with previous contexts
+argocd relogin
+
+# Reinitiates the login with password
+"argocd relogin --password YOUR_PASSWORD"
```
Comment (crenshaw-dev) [inline cmd/argocd/commands/relogin.go:92]:
```suggestion
argocd relogin --password YOUR_PASSWORD
```

File: docs/user-guide/commands/argocd_relogin.md [hunk 3]
```diff
@@ -15,7 +15,18 @@ argocd relogin [flags]
 ### Examples
 
 ```
-argocd relogin --password YOUR_PASSWORD
+  
+# Reinitiates the login with previous contexts
+argocd relogin
+
+# Reinitiates the login with password
+"argocd relogin --password YOUR_PASSWORD"
```
Comment (crenshaw-dev) [inline docs/user-guide/commands/argocd_relogin.md:23]:
```suggestion
argocd relogin --password YOUR_PASSWORD
```

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

**Review by crenshaw-dev — APPROVED** [review]

**Pull request merged**